x-pack/plugins/index_lifecycle_management/public/store/selectors/policies.js

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+
+import {
+  defaultHotPhase,
+  defaultWarmPhase,
+  defaultColdPhase,
+  defaultDeletePhase
+} from '../reducers/policies';
+import {
+  PHASE_HOT,
+  PHASE_WARM,
+  PHASE_COLD,
+  PHASE_DELETE,
+  PHASE_ROLLOVER_AFTER,
+  PHASE_ROLLOVER_AFTER_UNITS,
+  PHASE_ROLLOVER_ENABLED,
+  PHASE_ROLLOVER_MAX_AGE,
+  PHASE_ROLLOVER_MAX_AGE_UNITS,
+  PHASE_ROLLOVER_MAX_SIZE_STORED,
+  PHASE_ROLLOVER_MAX_SIZE_STORED_UNITS,
+  PHASE_NODE_ATTRS,
+  PHASE_FORCE_MERGE_ENABLED,
+  PHASE_FORCE_MERGE_SEGMENTS,
+  PHASE_PRIMARY_SHARD_COUNT,
+  PHASE_REPLICA_COUNT,
+  PHASE_ENABLED,
+  PHASE_ATTRIBUTES_THAT_ARE_NUMBERS,
+  MAX_SIZE_TYPE_DOCUMENT
+} from '../constants';
+import { getIndexTemplates } from '.';
+
+export const getPolicies = state => state.policies.policies;
+export const getSelectedPolicy = state => state.policies.selectedPolicy;
+export const getIsSelectedPolicySet = state => state.policies.selectedPolicySet;
+export const getSelectedOriginalPolicyName = state => state.policies.originalPolicyName;
+
+export const getSaveAsNewPolicy = state =>
+  state.policies.selectedPolicy.saveAsNew;
+
+export const getSelectedPolicyName = state => {
+  if (!getSaveAsNewPolicy(state)) {
+    return getSelectedOriginalPolicyName(state);
+  }
+  return state.policies.selectedPolicy.name;
+};
+
+export const getAllPolicyNamesFromTemplates = state => {
+  return getIndexTemplates(state).map(template => template.index_lifecycle_name).filter(name => name);
+};
+
+export const getPhases = state => state.policies.selectedPolicy.phases;
+export const getPhase = (state, phase) =>
+  getPhases(state)[phase];
+export const getPhaseData = (state, phase, key) => {
+  if (PHASE_ATTRIBUTES_THAT_ARE_NUMBERS.includes(key)) {
+    return parseInt(getPhase(state, phase)[key]);
+  }
+  return getPhase(state, phase)[key];
+};
+
+export const splitSizeAndUnits = field => {
```

Comment:
Doesn't look like this is used externally. Can we go through and remove `export` from all methods which are only for internal use?